lambda/postSchedulePeerTeachers/Scheduler/src/main/java/db/Person.java

```diff
@@ -115,18 +118,17 @@ public float getAvailabilityScore() {
     }
 
     public int getCurrentAssignments() {
-        return currentAssignments;
+        return sections.size();
     }
```

Comment:
This function name feels like it would return the actual assignments themselves rather than sections.size(), it could probably be named better

Reply:
Maybe getCurrentAssignmentsCount() or something